state/state.go

```diff
@@ -218,6 +219,26 @@ func (state *HelmState) DeleteCharts(helm helmexec.Interface) []error {
 	return nil
 }
 
+// FilterCharts allows for the execution of helm commands against a subset of the charts in the helmfile.
+// returns an error a chartName doesn't exist in the list of charts
+func (state *HelmState) FilterCharts(chartNames []string) error {
+	var filteredCharts []ChartSpec
+	for _, name := range chartNames {
+		missingName := true
+		for _, chart := range state.Charts {
+			if chart.Name == name {
```

Comment:
Nearly a nit but I wanted to confirm if, probably you're confusing release name vs chart name?
`ChartSpec.Name` stands for release name, whereas `ChartSpec.Chart` is the name of chart used to create the release.
Also see #25 for more context.

Reply:
I'm going to fix the terminology issue starting #35. So I'd fix it on my side if necessary.

Reply:
Your intention is to filter releases by release names rather than to filter charts by chart names, right?

Reply:
Absolutely, I see you've made things more consistent across the board (and more precise) in your PR. I'll rebase and fix my terminology.